src/libsyntax_ext/proc_macro_registrar.rs

```diff
@@ -170,11 +171,39 @@ impl<'a> Visitor for CollectCustomDerives<'a> {
                                   "derive mode defined twice in this crate");
         }
 
+        let proc_attrs: Vec<_> = item.attrs.iter()
+            .filter(|a| a.check_name("proc_macro_attributes"))
+            .filter_map(|a|
+              a.meta_item_list().or_else(|| {
```

Comment:
This spacing/indentation here is a bit difficult to read, could this be updated to follow the surrounding code?